lametro/events.py

```diff
@@ -97,6 +97,10 @@ def _merge_events(self, events):
             web_event = LAMetroWebEvent(web_event)
 
             if event.is_spanish:
+                try:
+                    assert spanish_events.get(event.key, None) == None
```

Comment:
i follow you, but i think something like `assert event.key not in spanish_events` is a little more straightforward. what do you think?

Reply:
yes, that makes sense. changing!